test/features/kogito_service_performance.feature

```diff
@@ -42,18 +42,13 @@ Feature: Kogito Service Performance
 #      | disabled | 10      | 320000   |
 
     @native
-    Examples: Native
```

Comment:
Accidentally removed?

Reply:
good catch. Might have been to fast to try to delete the `Native` string and deleted the rest ...